### PR TITLE
Added support for named schema

### PR DESCRIPTION
**Pull request by oxtoacart**
This adds support for organizing data into named schemas

- [x] Do the tests pass? Consistently?
- [x] Did this change improve test coverage?
- [x] Is the code in question being linted? If not, consider adding a linter step to CI. If yes, make sure the linter is happy.
- [x] Do we know how we’re going to deploy this?
- [x] Are we clear on what the support and maintenance impact is?
- [ ] Did you create new documentation? Is it accessible and in the right place?
- [ ] Has existing documentation been updated?
- [ ] Have you logged tickets for related technical debt with the label “techdebt”?

### CODE REVIEW

File: db/src/main/java/io/lantern/db/DB.kt [hunk 1]
```diff
@@ -103,7 +103,7 @@ abstract class Subscriber<T : Any>(id: String, vararg pathPrefixes: String) :
  */
 abstract class DetailsSubscriber<T : Any>(
     id: String,
-    private vararg val pathPrefixes: String
+    private vararg pathPrefixes: String
```
Comment (oxtoacart) [inline db/src/main/java/io/lantern/db/DB.kt:106]: drive-by, didn't need to be a val

**Review by markhor-sarwar — APPROVED** [review]
LGTM.

**Comment by oxtoacart** [comment]
Thanks @markhor-sarwar !
